app/controllers/hub/bulk_actions/change_assignee_and_status_controller.rb

```diff
@@ -73,10 +73,10 @@ def create_change_assignee_and_status_notifications!
         bulk_update = BulkTaxReturnUpdate.create!(
           tax_return_selection: @selection,
           assigned_user: @form.assigned_user,
-          status: status_action == BulkTaxReturnUpdate::KEEP ? nil : @form.status,
+          state: status_action == BulkTaxReturnUpdate::KEEP ? nil : @form.status,
           data: {
             assigned_user: assignment_action,
-            status: status_action
+            state: status_action
```

Comment:
should we also be updating the old `status` column so it's in sync with the new one? that appears to be what we're doing with `TaxReturn`s